src/main/fc/core.c

```diff
@@ -437,6 +438,10 @@ void tryArm(void)
         }
         imuQuaternionHeadfreeOffsetSet();
 
+#ifdef USE_GYRO_DATA_ANALYSE
```

Comment:
Another one. Should also be:
`#if (defined(USE_GYRO_DATA_ANALYSE) && !defined(SIMULATOR_BUILD))`

Basically we can't do anything related to the dynamic filter when in SIMULATOR_BUIILD (SITL).